docs/source/bundles.rst

```diff
@@ -47,15 +47,15 @@ process will invoke some custom bundle command and then write the data to a
 standard location that zipline can find. By default the location where ingested
 data will be written is ``$ZIPLINE_ROOT/data/<bundle>`` where by default
 ``ZIPLINE_ROOT=~/.zipline``. The ingestion step may take some time as it could
-involve downloading and processing a lot of data. This can be run with:
+involve downloading and processing a lot of data. You'll need a
+`Quandl <https://www.quandl.com/>`__ API key to ingest the default bundle. This can be run with:
```

Comment:
We might want to link to "Getting an API key" section of Quandl docs, https://docs.quandl.com/docs#section-authentication